tooling/helium-server/worker/index.js

```diff
@@ -65,3 +65,22 @@ function isAssetUrl(url) {
   const { pathname } = new URL(url);
   return pathname.startsWith('/assets/') || pathname.endsWith('manifest.json');
 }
+
+export default {
```

Comment:
All the changes related to worker is captured in df85686407aab373de757584b10bb92cd004f00f.

Was having trouble with testing these changes. I followed the steps below:

1. upgrade wrangler to latest version v2 (this is installed globally on my machine)
2. run wrangler build in `template-base` from the monorepo (the wrangler cli changes automatically from build to `wrangler publish --dry-run --outdir=dist`)

The last step failed for me, fyr, attached here is the output  log from the last step. Some of these errors seem strange, not sure if this is due to running the command in a monorepo setup. (had similar module resolution errors occur while upgrading the deps across 2 workspaces between `helium-server` and `template-base` .)
[wrangler_error.log](https://github.com/thoughtindustries/helium/files/9864622/wrangler_error.log)

Reply:
closing this thread.